jdaviz/configs/cubeviz/plugins/viewers.py

```diff
@@ -25,23 +25,44 @@ class WithSliceIndicator:
     def slice_component_label(self):
         return str(self.state.x_att)
 
+    @property
+    def slice_display_unit_name(self):
+        return 'spectral'
```

Comment:
I changed `slice_axis` to `slice_display_unit_name`, let me know if another name sounds more appropriate. Otherwise I can merge once CI passes (whatever can run right now at least).